include/nn/nex/berkeley.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <nn/types.h>
+
+namespace nn {
```

Comment:
You can use C++17 nested namespaces to make this more readable:
 
```suggestion
namespace nn::nex {
```